Computer-SpaceGames/Starship-Takeoff/cpp/src/game.cpp

```diff
@@ -0,0 +1,57 @@
+#include <string>
+#include "game.h"
```

Comment:
It's better practice to put the header file corresponding to the source file (i.e. `game.h`) first in the list of includes. This is because it makes sure that header file is self contained and doesn't contain any definitions that require other includes to come first. (Imagine the situation were this were a library that had users you don't know.)